frontend/assets/css/main.css

```
@@ -3,6 +3,7 @@
 @import url("./main/reset.css");
 @import url("./main/menu.css");
 @import url("./main/controls.css");
+@import url("./main/perf.css");
```

Stylelint (17.9.0)

[error] Expected "url("./main/perf.css")" to be ""./main/perf.css"" (import-notation)

(import-notation)